src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: max-content;
+  font-weight: 400;
+  padding: 32px $card-bottom-right-padding $card-bottom-right-padding;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    align-self: center;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0;
+    width: $card-title;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__rating {
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__amount {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__link {
+    box-sizing: border-box;
+    padding: 11px 0;
+    text-decoration: none;
+    text-align: center;
+    background: $hover-color;
+    border: 1px solid $hover-color;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: $white-color;
+  }
+
+  &:hover &__link {
+    background: $white-color;
+    color: $hover-color;
+  }
```

Comment:
maybe it's better to change the Button color when you move a mouse over the button, not over the card